frontend/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Pushy Notification</title>
 </head>
-<body onload="checkWebPushNotificationSupport()">
+<body onload="startWebPushNotificationFlow()">
     <div style="padding-left: 2%; font-family: sans-serif;">
         <h1 style="font-size: 70px;">Push'y' Notification</h1>
```

Comment:
I did not realise this earlier. But we usually have a separate repos for client side and server side code. That's how pushowl codebase is built and through this onboarding project you will get an initial idea of how the pushowl codebase is written. It's not an issue. Just mentioning, that our client side is a separate repo

Reply:
Ah okay, I will try to separate the repos if time permits before the demo